.coderabbit.yaml

```
@@ -0,0 +1,23 @@
+# .coderabbit.yaml
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)